### PR TITLE
added app implementation

### LINKED ISSUE #1: Initiate Spring Boot application
Initiate Spring Boot Gradle-based application.

Dependencies:
* spring-web
* spring-data
* in-memory DB

Implement `/greeting` API that returns "Hello".

### PR DESCRIPTION
**Pull request by andy-sam**
Fixes #1

### CODE REVIEW

File: web-banking/build.gradle [hunk 1]
```diff
@@ -0,0 +1,26 @@
+plugins {
+	id 'org.springframework.boot' version '2.5.1'
+	id 'io.spring.dependency-management' version '1.0.11.RELEASE'
+	id 'java'
+	id 'war'
```
Comment (alex-anakin) [inline web-banking/build.gradle:5]: please make it `jar`

File: web-banking/src/main/java/CodeSarge/webbanking/ServletInitializer.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package CodeSarge.webbanking;
+
+import org.springframework.boot.builder.SpringApplicationBuilder;
+import org.springframework.boot.web.servlet.support.SpringBootServletInitializer;
+
+public class ServletInitializer extends SpringBootServletInitializer {
```
Comment (alex-anakin) [inline web-banking/src/main/java/CodeSarge/webbanking/ServletInitializer.java:6]: please provide info on why the project needs this file

**Review by alex-anakin — CHANGES_REQUESTED** [review]
please implement `/greeting` API

File: web-banking/src/main/java/CodeSarge/webbanking/ServletInitializer.java (hunk 2 above)
Comment (andy-sam) [reply web-banking/src/main/java/CodeSarge/webbanking/ServletInitializer.java:6]: test

**Pull request closed without merging**